install.sh

```diff
@@ -134,12 +123,12 @@ else
 				echo "Indigo, the SubjuGator repository at an earlier date,  and all of the old"
 				echo "SubjuGator dependencies will need to be downloaded and installed."
 				echo -n "Do you still wish to proceed? [y/N] " && read RESPONSE
-				if ([ "$RESPONSE" = "Y" ] || [ "$RESPONSE" = "y" ]); then
+				if ([ "$RESPONSE" == "Y" ] || [ "$RESPONSE" == "y" ]); then
```

Comment:
If these changes are purely cosmetic, then there is something _very_ wrong with bash.